provider/lib/lib.go

```diff
@@ -33,6 +38,7 @@ func GetConfig(filepath string) ([]Config, error) {
 	if err != nil {
 		return nil, err
 	}
+	configs = append(configs, builtinConfig)
```

Comment:
basically made it so that builtin is -always- added to the configs returned by GetConfig